containers/Dockerfile.access-om2

```diff
@@ -7,9 +7,9 @@ WORKDIR $SPACK_ROOT
 
```

Comment:
Should the line above be `WORKDIR ${SPACK_ROOT}`? Sorry I couldn't select the correct line (line 6) in the review.

Reply:
Correct, good catch!